src/GraphQL/Validation/ValidationError.cs

```diff
@@ -36,7 +36,10 @@ public ValidationError(string originalQuery, string errorCode, string message, p
             params INode[] nodes)
             : base(message, innerException)
         {
-            Code = errorCode;
+            Code = ErrorInfoProvider.GetErrorCode(GetType());
+            if (Code != "VALIDATION_ERROR" && Code.EndsWith("_ERROR"))
+                Code = Code.Substring(0, Code.Length - 6);
+            Number = errorCode;
```

Comment:
Please change `errorCode` argument to `number`.

Reply:
Of course